### PR TITLE
Allow to display only specific countries

### PR DESCRIPTION
**Pull request by yauri-io**
Request of #16 

Adding a new prop 'onlyCountries'  for displaying only specific countries

### CODE REVIEW

File: src/vue-tel-input.vue [hunk 1]
```diff
@@ -222,7 +226,12 @@ export default {
         .filter(Boolean)
         .map(country => ({ ...country, preferred: true }));
 
-      return [...preferredCountries, ...allCountries];
+      let countryList = allCountries;
```
Comment (iamstevendao) [inline src/vue-tel-input.vue]: Are lines from 229 -> 232 deprecated? (already being used in `getCountryList()`)

**Comment by iamstevendao** [comment]
@yauri-io, Since you didn't get back, I have fixed it myself. It's now available in v2.0.12.
Thank you for your support!

File: src/vue-tel-input.vue (hunk 1 above)
Comment (yauri-io) [reply src/vue-tel-input.vue]: @iamstevendao sorry I missed the notification.....

**Comment by yauri-io** [comment]
@iamstevendao Sorry I missed the notification...
Thanks for the fix!

**Pull request closed without merging**